### DISCUSSION TITLE
Список "Смотрю" в поиске

### DISCUSSION THREAD

**Discussion by Cipher4444** [cawa-93/anime-library#405] [Новые идеи]
Есть ли возможность удалять из этого списка тайтл? Обычно они сами пропадают, когда их досматриваешь, но когда дропаешь, он остаётся навсегда и мешает.
![image](https://user-images.githubusercontent.com/92264790/142771771-1dba4719-28da-4daa-9a88-e618af9e331a.png)
![image](https://user-images.githubusercontent.com/92264790/142771852-81bae535-659d-4d85-94f4-c25db6fede9b.png)

**Comment by cawa-93** [comment]
Нет, сейчас такой возможности нет. Это хорошее замечание. Предложите Pull Request если хотите

**Reply by cawa-93** [reply]
Аниме пропадёт из списка если вы него не открывали больше 20-ти дней
